clang/docs/ReleaseNotes.rst

```diff
@@ -166,7 +174,9 @@ Non-comprehensive list of changes in this release
 
 New Compiler Flags
 ------------------
-- New option ``-fno-sanitize-annotate-debug-info-traps`` added to disable emitting trap reasons into the debug info when compiling with trapping UBSan (e.g. ``-fsanitize-trap=undefined``).
+- New option ``-fno-sanitize-debug-trap-reasons`` added to disable emitting trap reasons into the debug info when compiling with trapping UBSan (e.g. ``-fsanitize-trap=undefined``).
```

Comment:
> when compiling with trapping UBSan (e.g. ``-fsanitize-trap=undefined``).

This is somewhat misleading because by itself -fsanitize-trap=undefined doesn't enable UBSan (as  you and Anthony have noted in https://discourse.llvm.org/t/rfc-emit-a-warning-when-fsanitize-trap-is-passed-without-associated-fsanitize/87893)